package.json

```diff
@@ -140,6 +140,7 @@
     "@osd/std": "1.0.0",
     "@osd/ui-framework": "1.0.0",
     "@osd/ui-shared-deps": "1.0.0",
+    "@reduxjs/toolkit": "^1.6.2",
```

Comment:
welp looking at 2.x branch of observability https://github.com/opensearch-project/observability/blob/2.x/dashboards-observability/package.json#L19 this will cause a breakage for them. can we in this file just declare `^1.6.1` and resolve it in the desired version in the lock file?

Reply:
Sure thing. I dont think that the minor version downgrade will be an issue for D&D anyways. Want to know more about these versioning issues that we have to manually resolve each time though.